node-bootstrapper/app.go

```diff
@@ -56,6 +58,8 @@ func (a *App) run(ctx context.Context, args []string) error {
 			return errors.New("--provision-config is required")
 		}
 		return a.Provision(ctx, ProvisionFlags{ProvisionConfig: *provisionConfig})
+	case "monitor":
```

Comment:
I feel like there could be better names. Like `watch` or `wait` or something else?

Reply:
changed to provision-wait for now